### ISSUE TITLE
Comments on JSX not working properly

### ISSUE THREAD

**Issue by franciscolourenco** [babel/babel-sublime#331]
Commenting doesn't seem to be working correctly. 

Code:
```js
export default class MyComponent extends Component {
  render: () {
    return (
      <div>
        // <div>
          <div>
            
          </div>
        </div>
      </div>
    )
  }
}
```
Scope:
```
source.js 
meta.class.js 
meta.class.property.js 
meta.group.braces.curly.js 
meta.group.braces.round.js 
meta.jsx.js
```

Any ideas?

**Comment by aristidesfl** [comment]
This is caused by having the default `Javascript` package installed. The package is necessary for  JSON syntax highlighting. Is there a possible fix for this, or will the javascript comment preferences alway override the Babel ones?

**Comment by fr0m** [comment]
babel-sublime could provide JSON syntax highlight, so the Javascript package is not necessary.

**Comment by borela** [comment]
Try FJSX15 https://github.com/borela/naomi there will be no need to disable the core Javascript package.

**Comment by ellipticaldoor** [comment]
Any fix for this?

**Comment by brianfeister** [comment]
I'm looking for a fix for this as well

**Comment by borela** [comment]
Figured out how to make this work https://github.com/borela/naomi, it turns out it's not as easy as it looks and it required additional meta scopes and this monstrosity https://github.com/borela/naomi/blob/master/plugin/ToggleJsxComment.py (I'll clean up the code in a couple of days) to detect the correct comment type to use and to "feel right".

The latest versions already ship with the plugin, .

**Comment by Thom1729** [comment]
This should be fixed in v10.0.0-beta.1. See #379 for more information on the new release. Please report any problems with v10 in a new issue.
